corehq/apps/smsbillables/migrations/0010_add_index_to_log_id.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+
+        # Changing field 'SmsBillable.gateway_fee'
+        db.alter_column(u'smsbillables_smsbillable', 'gateway_fee_id', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['smsbillables.SmsGatewayFee'], null=True, on_delete=models.PROTECT))
+        # Adding index on 'SmsBillable', fields ['log_id']
+        db.create_index(u'smsbillables_smsbillable', ['log_id'])
+
+
+        # Changing field 'SmsBillable.usage_fee'
```

Comment:
I did not change anything but the `db_index=True`. I let South generate the migration automatically. I can manually remove these fields. Don't know what the best course of action is.

fyi, i think south will now handle `on_delete` differently when we upgrade: http://south.readthedocs.org/en/latest/releasenotes/0.7.4.html#on-delete-handling.